packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -808,7 +808,7 @@ const CFDPasswordModal = ({
                 i18n_default_text='Congratulations, you have successfully created your {{category}} <0>{{platform}}</0> <1>{{type}}</1> account.'
                 values={{
                     type: type_label,
-                    platform: platform === CFD_PLATFORMS.MT5 && is_pre_appstore ? 'MT5' : getCFDPlatformLabel(platform),
+                    platform: 'MT5',
```

Comment:
@sergei-deriv This modal is used for Derivx success modals also. We need to check if the platform is MT5 or derivx

Reply:
> @sergei-deriv This modal is used for Derivx success modals also. We need to check if the platform is MT5 or derivx

It it mean I have to change getCFDPlatformLabel function which must return 'MT5' instead of 'Deriv MT5', am I right?

Reply:
```suggestion
                 
                    platform: platform === CFD_PLATFORMS.MT5  ? 'MT5' : getCFDPlatformLabel(platform),
```